blackboxopt/optimizers/testing.py

```diff
@@ -84,11 +84,12 @@ def quadratic_function(p1):
 
     if issubclass(optimizer_class, MultiObjectiveOptimizer):
         evaluation = eval_spec.create_evaluation(
-            objectives={"loss": None, "score": None}
+            objectives={"loss": None, "score": None},
+            constraints={"constraint": 10.0},
```

Comment:
but do we have multiobjective **and** constrained optimizer?

Reply:
In principle that's a possibility that we should be ready for :relaxed: